### ISSUE TITLE
No contact information or guidelines for contributors

### ISSUE THREAD

**Issue by MagicFab** [OCA/e-learning#3]
## Module
OCA e-learning

## Describe the bug
There is no contact information in the blog article announcing the launch of this module:
https://odoo-community.org/blog/news-updates-1/learn-odoo-basics-with-oca-elearning-and-empower-your-new-odoo-customers-179

The link to GItHub from the e-learning website is broken too: [FIXED]
https://elearning.odooapps.ch/slides

The project page doesn't have contact or contributing information either:
https://github.com/OCA/e-learning

I even watched the OCA Days 2024 presentation about this and couldn't find any contact information either.
https://www.youtube.com/watch?v=tVziWqBGUy0

I'd love to help with English and Spanish.

**Comment by ben3474** [comment]
@MagicFab Great to know you are interested in contributing! For the english part we are in the process of publishing the version but nothing in spanish. We need to describe a little bit how to contribute for the translations and what are the expected deliverables.
The issue with regards to the Github link from the e-learning website is fixed. We will get back to you for the other parts.
About the missing info in the readme, there is an ongoing pull request that should fix the problem about the contacts...

**Comment by pedrobaeza** [comment]
@ben3474 @vdewulf Why this is not hosted in odoo-community.org instead? Is there any kind of paywall for extra content?

**Comment by vdewulf** [comment]
Hello @pedrobaeza 
You talk about this?
https://elearning.odooapps.ch/slides
This was simply created by Open Net team and we didn't discussed this further (I focused on the video hosted on oca youtube channel and the eLearning repository). Do you see it being a problem?
As far as I know, there is no paywall for extra content.

**Comment by pedrobaeza** [comment]
Yes,that's what I mean. We don't usually put links to external sites outside OCA, and do cross-posting things like publishing issues in OCA/e-learning for something that is outside OCA.

**Comment by vdewulf** [comment]
Well, the eLearning App is inside the OCA now.

Open Net was the original author of the App and they offered a simple way for people to see how it works by sharing a link to a database dedicated to that.

Do you feel it is a threat for the OCA to inform people that they can use the eLearning App on the Open Net website?

What would you suggest to mitigate the risk you see?

**Comment by pedrobaeza** [comment]
E-learning app is from Odoo, not from Open Net: https://github.com/odoo/odoo/tree/17.0/addons/website_slides, and it doesn't seem to have any extension.

The only "contribution" I see is 2 courses in French in the provided link. Am I seeing something wrong?

**Comment by vdewulf** [comment]
Yes that's what I meant, Open Net created the content for the basics on Odoo, and it will improved with more contributions in the future. The next steps are to translate the course in English and Spanish with new contributions.
You might think what you want, but it is a contribution.

**Comment by pedrobaeza** [comment]
Yes, but I insist: it should be hosted in OCA, not outside, and don't use OCA/e-learning, which is a repository for hosting OCA extension modules over Odoo e-learning system for posting things related to it.

What I want to highlight is that this is not an OCA GitHub contribution. It's other kind of contribution, and as so, it shouldn't be promoted as an OCA contribution nor using OCA things for its "management".

**Comment by MagicFab** [comment]
Sorry I didn't post the [announcement](https://odoo-community.org/blog/news-updates-1/learn-odoo-basics-with-oca-elearning-and-empower-your-new-odoo-customers-179) for context. This is a contribution to OCA by Open Net, I think it will be integrated into OCA eventually - that would be more coherent. Maybe @vdewulf can clarify that some time.
I'd like to understand why @pedrobaeza being from Tecnativa is so demanding and seems somehow threatened. If this is an OCA internal matter please discuss it outside this bug report.

**Comment by pedrobaeza** [comment]
@MagicFab, being a regular contributor and former OCA board member, I always try to preserve a clean contribution landscape not favoring any company nor having branded things, which it's what's happening here, redirecting out of OCA to a branded specific website.

**Comment by MagicFab** [comment]
@pedrobaeza OK, so maybe take this to the board and not in a public bug report ?

**Comment by pedrobaeza** [comment]
Why? Things should be open and transparent. This is an open source community.

**Comment by MagicFab** [comment]
So, open a new bug report instead of hijacking this one ? The issue here is lack of contact information, not governance.

**Comment by pedrobaeza** [comment]
I think this thread is the properly one, as:

1. You are asking for a module in your initial comment, but there's no extra module, as explained in my previous comment, and this leads to all my dissertations.
2. This issue tracker shouldn't be used for something related to something outside OCA.

**Comment by ben3474** [comment]
We have provided a server in order to support the contribution having been done. If the OCA can provide one, it is perfect, there is no problem, we can transfer the database.
We simply wanted to contribute to the functional track and thouht that this contribution was interesting. If I am not wrong, the project as it appears has been validated by the OCA board. Despite it is not code, it is also important and helpful for the community (including users).
@vdewulf @pedrobaeza I let you decide what you think is best for the OCA, not a problem for us. Don't want to generate problems.

**Comment by pedrobaeza** [comment]
Thanks for your predisposition @ben3474. Don't get me wrong, I have nothing against you or your company, and I appreciate your initiative. I just want to make sure that everything can be properly bootstrap for this to not be something that is reduced to the current status, and for this, we need an "agnostic" platform and rules for contributing to this content, which right now is not possible but with your intervention.

**Comment by MagicFab** [comment]
> I think this thread is the properly one, as:
> 
>     1. You are asking for a module in your initial comment, but there's no extra module, as explained in my previous comment, and this leads to all my dissertations.
> 
>     2. This issue tracker shouldn't be used for something related to something outside OCA.

Thanks for clarifying, it's indeed confusing.

**Comment by vdewulf** [comment]
Hello,
I discussed this matter with several Board Members.

We agree that in an ideal world, this module should:
* not mention the links to the Open Net platform hosting the eLearning modules (https://elearning.odooapps.ch/slides)(there are sentences mentioning this link).
* not be mentioned on the OCA [blog post](https://odoo-community.org/blog/news-updates-1/learn-odoo-basics-with-oca-elearning-and-empower-your-new-odoo-customers-179) promoting the new module 

The OpenNet platform ((https://elearning.odooapps.ch/slides) should be replaced by an equivalent platform hosted by the OCA, to which the links above should lead to.

As this module is available from version 17 and that the OCA instance is stil on version 14, we propose to keep this PR on hold until migration of the OCA instance, where this module might then be installed. The migration project will be scheduled in 2025. In the meantime, we are happy that OpenNet hosts the module allowing people to see a nice quick demo in one click (unlike the runboat that requires several clicks on Github interface and would refrain non technical users).

Thanks @pedrobaeza for your vigilance, thanks @ben3474 for your contribution here and willingness to adapt to the feedback received.

**Comment by pedrobaeza** [comment]
Sorry to insist, but you are all the time talking about a module, while there's no module involved here, except the Odoo core e-learning one (`website_slides`), which is already present in 14.0 as well, and in fact, in the OCA instance is already installed:

https://odoo-community.org/slides

**Comment by sbidoul** [comment]
@pedrobaeza I'm not sure I understand what you mean. In #1 there is a module. It is (mostly) a data module, but a module nevertheless. Assuming the other issues are resolved, as Virigine suggests, is your issue that this data module should be in a repo with a different name? Or something else?

> (website_slides) which is already present in 14.0 as well, and in fact, in the OCA instance is already installed:

That is a good point, and perhaps #1 could be re-targeted to 14.0 easily.

**Comment by pedrobaeza** [comment]
Thanks for the pointer, @sbidoul. It's the first time someone points to "code", and the PR title doesn't help either. Anyway, the module is mostly data (except 3 auxiliary fields), which can be entered in any instance without installing such module. Or am I missing something?

**Comment by vdewulf** [comment]
Hello again, I'm coming back to this topic.

I am sorry if this wasn't clear from the start that there was a module included in this PR, even if it's only data in XML fields.

The module allows to have the content ready in one click (installing the module) and avoids the need to copy/paste content from one instance to another. The business context for this module is to let Odoo Integrators install this module on their customers database, so that each customer can manager the onboarding of their members on the Odoo tool. They can use the eLearning module for other purposes and build a full training content customized for their needs afterwards, but the first bricks of the training would be available through this eLearning OCA module.

In the future, Open Net and other contributors can add new content (xml data + video on the OCA youtube channel) to make it more exhaustive than just the Odoo Basics, in this module or in new modules, depending on the functional area.

To come back to the issue with the link to Open Net platform, as there is the eLearning module installed on version 14, it might be easy to propose the module in v14 and install it on the OCA Odoo Database.

@ben3474 Would you be open to make this work and create a PR on version 14 with the same content, except for the link to the external Open Net website, and pointing to https://odoo-community.org/slides instead (where we will then install this module)?

**Comment by MagicFab** [comment]
Going back to the original problem reported, if someone can update on possible fixes, I'd appreciate it. Perhaps also open new bugs for governance/dev discussion, that would make it easier to track that.

**Comment by vdewulf** [comment]
Hello @MagicFab 

Here is the update on the initial issues you reported.

1. There is no contact information in the blog article announcing the launch of this module:
https://odoo-community.org/blog/news-updates-1/learn-odoo-basics-with-oca-elearning-and-empower-your-new-odoo-customers-179

Answer: there is an invitation to follow the link to this Github repo and to create an issue to make suggestions. This is the way we usually collaborate inside the OCA, trhough topics on Github, so we'll keep the blog post as is.

2. The link to GItHub from the e-learning website is broken too: [FIXED]
https://elearning.odooapps.ch/slides

3. The project page doesn't have contact or contributing information either:
https://github.com/OCA/e-learning
This page will be updated automatically when the first PR is merged, if I understand this correctly (I'm not a developer and not used to the internal process on OCA-Github).

Regarding the "governance" aspect, I created a separated issue:
https://github.com/OCA/e-learning/issues/5

**Comment by vdewulf** [comment]
Closing as the 3 points have been answered and another issue is coming for the Spanish translation contribution.

**Comment by MagicFab** [comment]
Thank you for all the follow ups ! Much appreciated and no doubt useful for everyone!
